src/udev/net/link-config.h

```diff
@@ -12,13 +12,13 @@
 typedef struct link_config_ctx link_config_ctx;
 typedef struct link_config link_config;
 
-typedef enum MACPolicy {
-        MACPOLICY_PERSISTENT,
-        MACPOLICY_RANDOM,
-        MACPOLICY_NONE,
-        _MACPOLICY_MAX,
-        _MACPOLICY_INVALID = -1
-} MACPolicy;
+typedef enum MACAddressPolicy {
+        MAC_ADDRESS_POLICY_PERSISTENT,
+        MAC_ADDRESS_POLICY_RANDOM,
+        MAC_ADDRESS_POLICY_NONE,
+        _MAC_ADDRESS_POLICY_MAX,
+        _MAC_ADDRESS_POLICY_INVALID = -1
+} MACAddressPolicy;
```

Comment:
this is a very good rename, in particular as MAC can also mean mandatory access control. This makes things a lot clearer! thanks!